EKF:
  robot_frame: "/base_footprint"
  optical_frame: "/camera_color_optical_frame"
  camera_info: "/camera/color/camera_info"
  rgb_topic: "/camera/color/image_raw"
  depth_aligned_topic: "/camera/aligned_depth_to_color/image_raw"
  depth_pointcloud_topic: "/depth_registered/points"
  rgb_detected: "/camera/color/image_detected"
  odom_topic: "/odom" 
  apriltag_marker_topic: "/apriltags" 